### ISSUE TITLE
Download Generated to many

### ISSUE THREAD

**Issue by jhaylee2016** [jerome-amp/Youtube-Downloader#5]
hey sir, this is an awesome script...but it is more awesome if its only generate a 1080. 720p 480  360 and 240 p...how to that...

**Comment by jerome-amp** [comment]
Hi,
I do not think I understand your request, if the goal is to filter the results, it is up to you to do it in your "view", you can do it easily with itags :

foreach($youtube-> videos as $video)
{
	if(in_array($video['itag'], array(137, 136, 135, 134, 133))) // 1080p 720p 480p 360p 240p
	{
		...
	}
}

**Comment by jhaylee2016** [comment]
yes sir you get my request...I want to generate these resolution 1080p 720p 480p 360p 240p links...but how to do with these i like to it to generate manually: for example



<a href="<?php echo $1080plinks; ?>">1080p Links</a>

<a href="<?php echo $720plinks; ?>">720p Links</a>

something like these...I want it to call manually the links generated...how can i do that..can you do the 1080p links sample code and I will do the rest..

thank you..

**Comment by jerome-amp** [comment]
I think you miss the index.php file : 
[https://github.com/jerome-amp/Youtube-Downloader/blob/master/index.php](https://github.com/jerome-amp/Youtube-Downloader/blob/master/index.php)
You already have the links and how to use the Youtube object.

**Comment by jhaylee2016** [comment]
yeah i get you sir but I don't want that I will echo in array...how to echo them manually:

foreach($youtube-> videos as $video)
{
if(in_array($video['itag'], array(137, 136, 135, 134, 133))) // 1080p 720p 480p 360p 240p
{
echo $video['url'][137];
}
}

how to pass them one by one...

**Comment by jerome-amp** [comment]
I update the files with the new img management, and change the key of video form autoincrement to itag.

So you can now do something like that now : $youtube->videos[137]['url'];

**Comment by jhaylee2016** [comment]
i get it..this update solve my problem...thank you so much...
